enhancements/proposals/1405-alternator/README.md

```diff
@@ -0,0 +1,182 @@
+# Support for ScyllaDB Alternator
+
+## Summary
+
+This proposal aims to introduce proper support for enabling Alternator for ScyllaClusters 
```

Comment:
```suggestion
This proposal aims to introduce proper support for enabling Alternator for Scylla Clusters 
```

Reply:
we are intentionally putting those together as it's the exact CRD name